cmake/Platform.cmake

```diff
@@ -96,6 +96,9 @@ endif()
 if(EMSCRIPTEN)
   add_compile_options(-fdeclspec)
 
+  # Enable web simd
+  add_compile_options(-msimd128)
```

Comment:
did this actually work? cos I had issues with rust

Reply:
I should try running a test build yes, but at least compiles the shard tests yes

Reply:
Yeah. Although for me was not even building.